frontend/src/components/common/LogRecordsTable.tsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import {
+  Table,
+  TableContainer,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from "@chakra-ui/react";
+import { LogRecord } from "../../types/LogRecordTypes";
+
+type Props = {
+  logRecords: LogRecord[];
+};
+
+const LogRecordsTable = ({ logRecords }: Props): React.ReactElement => {
+  return (
+    <div>
+      <TableContainer marginTop="12px" height="70vh" overflowY="scroll">
```

Comment:
on my screen, seems like there's a large gap when the user selects 10 items:
![image](https://github.com/uwblueprint/supportive-housing/assets/57375371/3c025105-cac0-4cb6-95b3-b217f1992198)

Reply:
(Presumably) will be dealt with in https://github.com/uwblueprint/supportive-housing/issues/97